R/release.R

```diff
@@ -111,9 +110,6 @@ release_checklist <- function(version, on_cran) {
     todo("`devtools::build_readme()`", has_readme),
     todo("`devtools::check(remote = TRUE, manual = TRUE)`"),
     todo("`devtools::check_win_devel()`"),
-    todo("`rhub::check_for_cran()`"),
-    todo("`rhub::check(platform = 'ubuntu-rchk')`", has_src),
```

Comment:
rchk (and maybe sanitizers) is one i do find helpful, because IIRC these checks don't get run until _after_ CRAN accepts your package. So it can be accepted and then you can get hit with a 2 week email due to an rchk issue, and this check would have helped you avoid that.

That is how I'm remembering this, but if I'm wrong about that and rchk is run as a pre-check by CRAN, then i wouldn't mind if this was removed

Reply:
I think my personal experience lately has been that I didn't actually get the info I needed from these. Either I got a PREPERROR and no information, r-hub's rchk results don't match CRAN's, or all the rchk results were actually all about cpp11 and not, e.g., readxl's code.

The initial item "check CRAN results" helps a bit with this, because you would presumably see if you had pre-existing rchk issues.

So, yeah, we are technically losing something. But I'm not sure it matters in practice. I could add these back with the condition being `has_src && !on_cran` to partially mitigate.

Reply:
We can just remove them, nbd